lib/pkgr/env.rb

```diff
@@ -2,6 +2,10 @@ module Pkgr
   class Env
     attr_reader :variables
 
+    def self.from_export(file)
+      new File.read(file).split("\n").map { |line| line.sub!("export", "") }
```

Comment:
@benlangfeld can you put parenthesis around the `new` keyword? Also, I suspect it should be `line.sub(/^export\s+/, "")` no?